### PR TITLE
Show video controls immediately on mouse move

### LINKED ISSUE #208: UI responsiveness
- [x] Disabling download of a torrent doesn't update UI immediately
- [x] Dropping a large file to seed takes *seconds* to show up in the UI.
  - We should show it right away, and not wait for the `'torrent'` event which only happens *after* the whole file has been hashed.
- [ ] Mouseover events take up to 1 second to kick in because they're not handled right away but only on `update()`. The right way to handle them is immediately, but [throttled](https://npmjs.com/package/throttleit).

### PR DESCRIPTION
**Pull request by dcposch**
Fixes #208 

@feross LMK if you think there's a cleaner way to do this

Also avoids loading all of the cast modules (chromecast/airplay/dnla) the first time you play/pause/seek a video when you're not casting.

### CODE REVIEW

File: renderer/index.js [hunk 1]
```diff
@@ -285,7 +286,7 @@ function dispatch (action, ...args) {
   }
 
   // Update the virtual-dom, unless it's just a mouse move event
-  if (action !== 'mediaMouseMoved') {
+  if (action !== 'mediaMouseMoved' || showOrHidePlayerControls()) {
```
Comment (feross) [inline renderer/index.js]: What is this for?

File: renderer/index.js [hunk 2]
```diff
@@ -324,16 +325,26 @@ function changeVolume (delta) {
   setVolume(state.playing.volume + delta)
 }
 
+// TODO: never called. Either remove or make a volume control that calls it
```
Comment (grunjol) [inline renderer/index.js]: Started working on it today, ETA tomorrow night

Comment (feross) [inline renderer/index.js]: cool!

**Comment by feross** [comment]
This is fine for now, but I think we can clean this up better by handling whether to show/hide the control mostly in CSS. I'll do that later.

**Pull request merged**